platformio.ini

```diff
@@ -144,7 +130,6 @@ check_flags =
   ; The actual checks are defined in .clang-tidy.
   clangtidy: --checks='-*' --extra-arg-before=-xc++ --extra-arg-before=-std=c++17
 check_patterns =
-  common/include
```

Comment:
Why this change?

Reply:
We don't have this directory, having this here poisons vs code include resolution. It has to be removed some time, this time is as good as any